.github/workflows/main-tier0.yml

```
@@ -12,30 +12,11 @@ on:
-        with:
-          name: analysis-tests-output
-          include-hidden-files: true
-          path: analysis/tmp_output
+    uses: konveyor/ci/.github/workflows/global-ci-bundle.yml@main
+    with:
+      operator_bundle: quay.io/konveyor/tackle2-operator-bundle:latest
+      api_tests_ref: main
+      run_api_tests: true
+      run_ui_tests: false
+      api_tests_tiers: DEBUG=1 make test-tier0
+  
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)